# Smith 06
The verbatim claim: "λελωφήκαμεν: rather rare, used by Thuc. with reference to sicknesses and grave misfortunes."

Our interpretation: 1) rather rare = the odds of picking a verb with this lemma (λωφάω) from all verbs in the Perseus corpus is not very likely and 2) λελωφήκαμεν usually occurs in a context when Thucydides is discussing “sicknesses and grave misfortunes.”

## 0) Requirements

To run and tweak this analysis, you will need the following:
- the [repository](https://github.com/lepidopterane-atsmith/tompkinsia) this file exists in
- the [pandas](https://pandas.pydata.org/docs/getting_started/index.html#installation) library

## 1) Rarity

First, let's import some necessary libraries and look at **all_verb_lemmata.csv,** which we can see is a list of verb lemmas and the number of times they occur in the Perseus corpus. 

In [2]:
import pandas as pd

lemmata = pd.read_csv('../all_verb_lemmata.csv')
lemmata.sort_values('count', ascending=False)[:20]

lemma   count
1050       εἰμί  209433
11096      λέγω   78161
45165  γίγνομαι   65809
21008       ἔχω   60845
25204     ποιέω   36583
49363      φημί   35067
49264       δέω   23717
42928     δοκέω   18658
14293      ὁράω   17522
42549   λαμβάνω   16627
45350     καλέω   14470
28383    δίδωμι   13962
50211   δύναμαι   13013
18255      χράω   12790
50271  βούλομαι   12462
16735      οἶδα   12020
13209   ἔρχομαι   11611
20480     φαίνω   11441
12879      φέρω   10231
42300    ὑπάρχω   10169

These are the 20 most common lemmata in our lemma group. To evaluate how unlikely it is to pick λωφάω from this group, we should have other odds to compare it against. Calculating the same odds for the most common lemma (εἰμί), the 20th most common lemma (ὑπάρχω), and maybe one least common lemma should be a good set to help us see how rare or not rare λωφάω is. Let's pick the least common lemma:

In [3]:
lemmata.sort_values('count', ascending=False)[-5:]

lemma  count
50471    διατμάζω      1
50470  ὑποπέτομαι      1
50469  ἀνταναμένω      1
50467   ἀπκαλύπτω      1
50464   διάπλομαι      1

διάπλομαι will do nicely, a single instance in a large set of lemmata is definitely rare.

Odds are calculated the following way: **undesirable outcomes / desirable outcomes = x** where we then say there is a 1 in x chance of the desirable outcome. In our case, let's say that desirable outcomes are the counts associated with each lemma, and the undesirable outcomes can be calculated by subtracting each desirable outcome from the sum of everything in the **count** column.  

In [4]:
total_count = int(lemmata['count'].sum())
desirable_outcomes, undesirable_outcomes = [],[]
verbs = ["εἰμί", "ὑπάρχω", "λωφάω", "διάπλομαι"]

for i in verbs:
    desirable_outcomes.extend(lemmata[lemmata['lemma']== i]['count'].values)
desirable_outcomes = [int(i) for i in desirable_outcomes]

for d in desirable_outcomes:
    undesirable_outcomes.append(total_count-d)

for e in range(4):
    odds = undesirable_outcomes[e]/desirable_outcomes[e]
    print("The odds of picking",verbs[e],"from the lemmata group is 1 in",odds,".",sep=" ")


The odds of picking εἰμί from the lemmata group is 1 in 9.224367697545278 .
The odds of picking ὑπάρχω from the lemmata group is 1 in 209.57331104336708 .
The odds of picking λωφάω from the lemmata group is 1 in 41178.230769230766 .
The odds of picking διάπλομαι from the lemmata group is 1 in 2141319.0 .


The odds of picking λωφάω are about 200 times less than picking ὑπάρχω, and since 1 in ~40,000 is less than 1 in 10,000, which can also be thought of as the desired outcome representing .01% of a set, one could state that the occurrence of λωφάω is in fact uncommon enough to be rare. Now that we have established that λωφάω is rather rare, let us look at the second piece of this claim. 

## 2) Sicknesses and Grave Misfortunes

First, we will look up where λωφάω occurs in Thucydides' works using our bespoke query engine (under development).

In [5]:
from simple_query import create_engine_from_files, word_search

query_engine = create_engine_from_files(urns=['0003-001', '0003-002'])
results = word_search(query_engine, "λωφάω")
results

URNs in system:  2
loading  2  items...
6055
λωφάω


Form  Lemma Part of Speech        Case    Number    Tense  \
0    λωφήσαντα  λωφάω           verb  accusative  singular   aorist   
1      λώφησιν  λωφάω           verb              singular  present   
2  λελωφήκαμεν  λωφάω           verb                plural  perfect   
3    λωφήσειαν  λωφάω           verb                plural   aorist   

         Mood   Voice Relation  Sentence ID  Word ID Document URN  
0              active      ATR         1281  7271687     0003-001  
1  indicative  active       CO         3038  7318243     0003-001  
2  indicative  active      OBJ         4150  7348288     0003-001  
3    optative  active     PRED         5273  7382417     0003-001

As you can see, there are four instances in the History, which we will look at one by one. We can open up the History's [treebank file](https://github.com/lepidopterane-atsmith/tompkinsia/raw/refs/heads/main/comment_01/data/xml/0003-001.xml) and use cmd/ctrl + F to search for `sentence id=`**"Sentence ID in the table above"** and match those to subdoc notes and the first four words, as seen in the following subsections. This makes sure we are looking at the right instance, and you can do something similar with the Word ID.

For each of these four instances, we will take a closer look and assess whether each instance supports or invalidates this assumption. Each passage is accompanied by translations from Charles Foster's early 1900s editions.

### Instance 1: 2.49.4, Sentence ID 1281, λύγξ τε τοῖς πλέοσιν

> λύγξ τε τοῖς πλέοσιν ἐνέπιπτε κενή, σπασμὸν ἐνδιδοῦσα ἰσχυρόν, τοῖς μὲν μετὰ ταῦτα λωφήσαντα, τοῖς δὲ καὶ πολλῷ ὕστερον.

> ...and in most cases ineffectual retching followed producing violent convulsions, which sometimes abated directly, sometimes not until long afterwards. 

This passage documents symptoms of the Athenian Plague, which was happening at the same time as the Peloponessian War. λωφήσαντα is certainly used in the context of a sickness. 

### Instance 2: 4.81.2, Sentence ID 3038, τό τε γὰρ παραυτίκα
> τό τε γὰρ παραυτίκα ἑαυτὸν παρασχὼν δίκαιον καὶ μέτριον ἐς τὰς πόλεις ἀπέστησε τὰ πολλά, τὰ δὲ προδοσίᾳ εἷλε τῶν χωρίων, ὥστε τοῖς Λακεδαιμονίοις γίγνεσθαι ξυμβαίνειν τε βουλομένοις, ὅπερ ἐποίησαν, ἀνταπόδοσιν καὶ ἀποδοχὴν χωρίων καὶ τοῦ πολέμου ἀπὸ τῆς Πελοποννήσου λώφησιν· ἔς τε τὸν χρόνῳ ὕστερον μετὰ τὰ ἐκ Σικελίας πόλεμον ἡ τότε Βρασίδου ἀρετὴ καὶ ξύνεσις, τῶν μὲν πείρᾳ αἰσθομένων, τῶν δὲ ἀκοῇ νομισάντων, μάλιστα ἐπιθυμίαν ἐνεποίει τοῖς Ἀθηναίων ξυμμάχοις ἐς τοὺς Λακεδαιμονίους.

> For, at the present crisis, by showing himself just and moderate in his dealings with the cities he caused most of the places to revolt, and secured possession of others by the treachery of their inhabitants, so that when the Lacedaemonians wished to make terms with Athens, as they did ultimately, they had places to offer in exchange for places they wished to recover and were able to secure for the Peloponnesus a respite from the war; and in the later part of the war, after the events in Sicily, it was the virtue and tact which Brasidas had displayed at this time—qualities of which some had had experience, while others knew of them by report—that did most to inspire in the allies of the Athenians a sentiment favourable to the Lacedaemonians.

Foster translates λώφησιν as respite, as it is actually from the noun λώφησις and not the verb λωφάω. Technically, this does not count as an instance of λωφάω and will not be assessed, though it should be known that this word was mislabeled as a form of λωφάω in the treebanks being searched over for this part of the analysis.

### Instance 3: 6.12.1,  Sentence ID 4150, ‘καὶ μεμνῆσθαι χρὴ ἡμᾶς

> ‘καὶ μεμνῆσθαι χρὴ ἡμᾶς ὅτι νεωστὶ ἀπὸ νόσου μεγάλης καὶ πολέμου βραχύ τι λελωφήκαμεν, ὥστε καὶ χρήμασι καὶ τοῖς σώμασιν ηὐξῆσθαι· 

> And we should remember that we have but lately recovered somewhat from a great pestilence and war, so as to recruit our strength both in money and in men; 

Nicias is purposely invoking the Athenian recovery from the Athenian Plague, using the λελωφήκαμεν this comment and its claim are anchored on, in the context of a sickness. 

### Instance 4: 7.77.3, Sentence ID 5273, τάχα δὲ ἂν καὶ

> ἀνθ’ ὧν ἡ μὲν ἐλπὶς ὅμως θρασεῖα τοῦ μέλλοντος, αἱ δὲ ξυμφοραὶ οὐ κατ’ ἀξίαν δὴ φοβοῦσιν. τάχα δὲ ἂν καὶ λωφήσειαν· ἱκανὰ γὰρ τοῖς τε πολεμίοις ηὐτύχηται, καὶ εἴ τῳ θεῶν ἐπίφθονοι ἐστρατεύσαμεν, ἀποχρώντως ἤδη τετιμωρήμεθα.

> Wherefore, in spite of all, my hope for the future is still confident, and our calamities do not frighten me as much as they might well have done. Perhaps they may even abate; for our enemies have had good fortune enough, and if we have roused the jealousy of any of the gods by our expedition we have already been punished sufficiently. 

Nicias is using what he knows of rhetoric and past Athenian might to reinvigorate Athenian forces that have recently experienced a great loss. His language is somewhat ambiguous as he documents times when in the face of some calamity Athenians find a way forward nonetheless, but this seems more entrenched in the "grave misfortunes" part of the claim than the "sicknesses" part.

Smith says that Thucydides usually uses this in the context of grave misfortunes and sicknesses, leading the uncritical reader to assume that he uses this lemma more than three times, and that there may be some instances that occur not in the context of sickness but in the former context, along with a handful of instances that do not mention sickness or misfortune. The analysis in this notebook shows that Thucydides uses this word thrice, always in the context of the Athenian Plague, except for when Nicias is discussing grave misfortunes past and present in 7.77.3, and there is no direct mention of the plague. While this could validate Smith's comment, it would be remiss not to remark that the wording of the claim misleads the author to believe that this is a more frequent and less specific pattern than it is. Combined with earlier confirmation that λωφάω is in fact rare, we find here that these claims hold up, even if the wording on one is somewhat misleading.